app/views/posts/_list_item.haml

```diff
@@ -3,23 +3,23 @@
   - replies_count = post.reply_count
   %td.post-completed.vtop{class: [klass, (post.completed? ? 'post-complete' : 'post-incomplete')]}
     - if post.completed?
-      = image_tag "icons/book.png", class: 'vmid', title: "Thread Complete"
+      = image_tag "icons/book.png", class: 'vmid', title: "Thread Complete", alt: 'Complete'
     - elsif post.on_hiatus?
-      = image_tag "icons/hourglass.png", class: 'vmid', title: "Thread On Hiatus"
+      = image_tag "icons/hourglass.png", class: 'vmid', title: "Thread On Hiatus", alt: 'Hiatused'
     - elsif post.abandoned?
-      = image_tag "icons/book_grey.png", class: 'vmid', title: "Thread Abandoned"
+      = image_tag "icons/book_grey.png", class: 'vmid', title: "Thread Abandoned", alt: 'Abandoned'
     - if post.access_list? || post.private?
-      = privacy_icon(post.privacy)
+      = privacy_icon(post.privacy, true)
     - if !current_user.try(:hide_warnings) && post.has_content_warnings?
-      = image_tag "icons/exclamation.png", class: 'vmid', title: "Content Warning: " + post.content_warnings.pluck(:name).join(', '), alt: '!'
+      = image_tag "icons/exclamation.png", class: 'vmid', title: "Content Warning: " + post.content_warnings.pluck(:name).join(', '), alt: 'Has Warnings'
   %td.post-subject.vtop{class: klass}
     - if logged_in?
       - if unread_post?(post, unread_ids) || (@show_unread && !opened_post?(post, opened_ids))
         = link_to unread_path(post) do
-          = image_tag unread_img, class: 'vmid', title: 'First Unread'
+          = image_tag unread_img, class: 'vmid', title: 'First Unread', alt: 'First Unread'
       - elsif @show_unread
         = link_to unread_path(post) do
-          = image_tag lastlink_img, class: 'vmid', title: 'First Unread'
+          = image_tag lastlink_img, class: 'vmid', title: 'First Unread', alt: ''
```

Comment:
I think this happens when the post is totally read but we're showing a link anyway (like the posts owed page) – this should have an alt too, right?

Reply:
This was one of the ones I was more uncertain on, because it seemed to straddle the line between functional and decorative, but if you think it should have alt text I can do that. I just... need alt text for it :P

Reply:
I'd go with `First Unread` or `Jump to First Unread`, same as the other, even if that doesn't technically describe it perfectly. Alternatively, `Jump to end` or maybe `Jump to end (read)` would be good?

Reply:
(I think basically every image inside a link without accompanying text is functional, fwiw)